orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/AdminController.groovy

```diff
@@ -74,6 +87,35 @@ class AdminController {
     forceExecutionCancellationCommand.forceCancel(executionType, executionId, canceledBy)
   }
 
+
+  @PostMapping(value = "/executions/")
+  @ResponseStatus(HttpStatus.CREATED)
+  Map<String, String> createExecution(@RequestBody Execution execution) {
+
+    if (front50Service && !front50Service.get(execution.application)) {
+      log.warn('No application exists with name: ' + execution.application)
+    }
+
+    try {
+      executionRepository.retrieve(execution.type, execution.id)
+      log.warn('Execution found with id: []', execution.id)
```

Comment:
`warn` seems like the wrong logging level? `debug` maybe? (or perhaps it's not even relevant given it's going to raise an exception immediately)